geoapi/src/main/java/org/opengis/metadata/acquisition/Objective.java

```diff
@@ -98,7 +98,7 @@
      *
      * @return events associated with objective completion.
      */
-    @UML(identifier="objectiveOccurence", obligation=MANDATORY, specification=ISO_19115_2)
```

Comment:
This typo exists in ISO 19115-2:2009, and the 2018 revision still have it. So the annotation can not be changed here since it reflects the UML published by ISO. But by coincidence, we are right now in a period where ISO is collecting comments for their 2018 revision. I have sent an email notifying this typo. I'm not sure they will fix it however since it would be an incompatible change with previous edition. I will add a comment in GeoAPI saying that this is a known misspelling.